pom.xml

```diff
@@ -63,7 +63,6 @@
         <joda.version>2.9.9</joda.version>
         <jsr353-api.version>1.0</jsr353-api.version>
         <jsr353-ri.version>1.0.4</jsr353-ri.version>
-        <jnr-ffi.version>2.0.9</jnr-ffi.version>
```

Comment:
I would suggest that we declare both dependencies for the reasons outlined above, with maybe a comment here to remind us that jnr-posix and jnr-ffi must have compatible versions.